testing/integration/diagnostics_test.go

```diff
@@ -20,11 +20,14 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
```

Comment:
Above you where talking about yaml.v3 but here you import yaml.v2? Which one is being used here? We should unify our yaml usage to a single library.

Reply:
I've updated this and all the integration test uses, and have made https://github.com/elastic/elastic-agent/issues/5750 to do the rest